### ISSUE TITLE
Add information around temp usage personal account for ADLS connection

### ISSUE THREAD

**Issue by Shuwi75** [MicrosoftDocs/powerbi-docs#4169]
Several customers are concerned around the usage of a personal account to create the connection between the PBI workspace and the ADLS store (user has to click on the 'Connect to Azure' button). The concern is that when this user changes role or leaves the AAD, the ADLS connection or stored dataflows get disturbed/ stop functioning. 

I have confirmation from PBI CAT PM this will not be the case, the personal account is used only once to set the initial connection, after that these permissions get replaced by a service account in the backend. " These permissions are used to grant the PBI Service account rights to the storage account to read and write data. After this is configured, the original user is no longer needed to keep the connection active. You can validate this by checking the permissions that PBI service has on your test storage account. " 
This will make sure everything keeps functioning when the initial user is removed from AAD, or the workspace. 

It might be a good idea to add this information to this page, taking away potential concerns from customers around this subject. 

Kind regards,
Jeroen

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 57f28e6c-a22d-484b-6599-9b17e6c011f0
* Version Independent ID: 307cb997-5d54-a43c-f278-5247769e682b
* Content: [Configuring dataflow storage to use Azure Data Lake Gen 2 - Power BI](https://learn.microsoft.com/en-us/power-bi/transform-model/dataflows/dataflows-azure-data-lake-storage-integration)
* Content Source: [powerbi-docs/transform-model/dataflows/dataflows-azure-data-lake-storage-integration.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/main/powerbi-docs/transform-model/dataflows/dataflows-azure-data-lake-storage-integration.md)
* Service: **powerbi**
* Sub-service: **pbi-dataflows**
* GitHub Login: @davidiseminger
* Microsoft Alias: **davidi**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @Shuwi75. This issue has been assigned to the doc author for review.

**Comment by davidiseminger** [comment]
Thanks for your comments, I've added that clarification to the article. Much appreciated.
